doc/nrf/libraries/modem/modem_key_mgmt.rst

```diff
@@ -57,6 +57,82 @@ The PEM format looks like this::
 
 See the comprehensive `tutorial on SSL.com`_ for instructions on how to convert between different certificate formats and encodings.
 
+Usage
+*****
+
+The different types of credentials supported by the library are defined by the :c:enum:`modem_key_mgmt_cred_type` enum.
+
+The following code snippet shows how to write a CA chain certificate to the modem:
+
+.. code-block:: c
+
+   int err;
+   static const char cert[] = {
+           #include "YourCert.pem.inc"
+   };
+
+   err = modem_key_mgmt_write(<YOUR_SEC_TAG>, MODEM_KEY_MGMT_CRED_TYPE_CA_CHAIN, cert, sizeof(cert));
+   if (err) {
+           printk("Failed to provision certificate, err %d\n", err);
+   }
+
+The following code snippet shows how to check if a CA chain certificate exists in the modem:
+
+.. code-block:: c
+
+   int err;
+   bool exists;
+
+   err = modem_key_mgmt_exists(<YOUR_SEC_TAG>, MODEM_KEY_MGMT_CRED_TYPE_CA_CHAIN, &exists);
+   if (err) {
+           printk("Failed to check if credential exists");
+           return;
+   }
+
+   if (exists) {
+           printk("Credential exists in the modem");
+   } else {
+           printk("Credential does not exist in the modem");
+   }
+
+The following code snippet shows how to check if the CA chain certificate stored in the modem is the same as another CA chain certificate:
+
+.. code-block:: c
+
+   int mismatch;
+   static const char cert[] = {
+           #include "YourCert.pem.inc"
+   };
+
+   mismatch = modem_key_mgmt_cmp(<YOUR_SEC_TAG>, MODEM_KEY_MGMT_CRED_TYPE_CA_CHAIN, cert, sizeof(cert));
+   if (mismatch) {
+           printk("Certificate mismatch\n");
```

Comment:
nit: some prints have `\n` and some don't. either is fine but align for consistency

Reply:
Added `\n` to all.